src/github.com/matrix-org/dendron/proxy/paths.go

```diff
@@ -0,0 +1,146 @@
+package proxy
+
+// Note that this file was generated by running `python dendron/scripts/generate_proxy_paths.py`.
+// It should not be edited by hand.
+
+import "regexp"
+
+type regexpAndString struct {
+	re *regexp.Regexp
+	s  string
+}
+
+var (
+	constantPaths = make(map[string]bool)
```

Comment:
If you're using the map as a set, a bool means that you can do either of:

```go
if ok := m["val"]

or

if _, ok := m["val"]
```

and they end up being the same (because default values), which is somewhat of a convention.

Conversely, for channels, if you're using a channel as a signal with no data, the runtime is mildly optimised for using struct{}.